libs/base/ui/structure/text/text.def.ts

```diff
@@ -1,6 +1,32 @@
-import { componentDef } from '@spryker-oryx/utilities';
+import { componentDef, screenCss } from '@spryker-oryx/utilities';
+import { css } from 'lit';
+
+const textScreenStyles = screenCss({
```

Comment:
Should these be in def file?

Reply:
This has also dropped from 1.4 onwards with the latest refactor of this PR.